packages/field-base/test/clear-button-mixin.test.js

```diff
@@ -135,6 +135,12 @@ const runTests = (defineHelper, baseMixin) => {
       expect(input.value).to.equal('foo');
     });
 
+    it('should not clear value on Esc when readonly is true', () => {
+      element.readonly = true;
```

Comment:
```suggestion
      element.clearButtonVisible = true;
      element.readonly = true;
```
Clear buttons needs to be shown, otherwise clearing with Escape is not possible.